include/bluetooth/iso.h

```diff
@@ -34,6 +34,17 @@ extern "C" {
 				  BT_HCI_ISO_HDR_SIZE + \
 				  BT_HCI_ISO_DATA_HDR_SIZE)
 
+/** @def BT_ISO_SDU_BUF_SIZE
+ *
+ *  @brief Helper to calculate needed buffer size for ISO SDUs.
+ *         Useful for creating buffer pools.
+ *
+ *  @param mtu Required ISO SDU size
+ *
+ *  @return Needed buffer size to match the requested ISO SDU MTU.
+ */
+#define BT_ISO_SDU_BUF_SIZE(mtu) (BT_ISO_CHAN_SEND_RESERVE + (mtu))
+
```

Comment:
@Thalley By the looks of it there is a bug in ISO when hci_raw is used. Since this is using CONFIG_BT_HCI_RESERVE instead of BT_BUF_RESERVE (i.e not including CONFIG_B_HCI_RAW_RESERVER).

In my opinion this would best be solved by having an BT_BUF_ISO_SIZE in bluetotoh/buf.h
Defined as: 
`#define BT_BUF_ISO_SIZE(size) BT_BUF_SIZE(BT_HCI_ISO_HDR_SIZE + BT_HCI_ISO_DATA_HDR_SIZE + (size))`
And:
`#define BT_ISO_SDU_BUF_SIZE(mtu) BT_BUF_ISO_SIZE(mtu))`
`#define BT_ISO_CHAN_SEND_RESERVE BT_BUF_ISO_SIZE(0)`

That way the base handling of reserved HCI bytes are all using the same definition.